langchain4j-google-ai-gemini/src/main/java/dev/langchain4j/model/gemini/GeminiAiChatLanguageModel.java

```diff
@@ -0,0 +1,254 @@
+package dev.langchain4j.model.gemini;
```

Comment:
```suggestion
package dev.langchain4j.model.googleai;
```

major: to be consistent with `dev.langchain4j.model.vertexai`